contractcourt/channel_arbitrator.go

```diff
@@ -994,7 +994,14 @@ func (c *ChannelArbitrator) stateStep(
 		if err := c.cfg.PublishTx(closeTx, label); err != nil {
```

Comment:
Hmm not sure, I did not include it there because we are not returning an error there just logging that we were not able to broadcast the closing transaction which seems right?